msmbuilder/preprocessing/timeseries.py

```diff
@@ -0,0 +1,61 @@
+from scipy.signal import lfilter, lfilter_zi, filtfilt, butter
+
+from .base import MultiSequencePreprocessingMixin
+
+class Butterworth(MultiSequencePreprocessingMixin):
+
+    def __init__(self, width, order=3):
+        """Smooth time-series data using a zero-delay Butterworth filter.
+
+        Parameters
+        ----------
+        width : int
+            This acts very similar to the window size in a moving average
+            smoother. In this implementation, the frequency of the low-pass
+            filter is taken to be two over this width, so it's like
+            "half the period" of the sinusiod where the filter starts
+            to kick in. Must be an integer greater than one.
+        order : int, optional, default=3
+            The order of the filter. A small odd number is recommended. Higher
+            order filters cutoff more quickly, but have worse numerical
+            properties.
+
+        References
+        ----------
+        .. [1] "FiltFilt". Scipy Cookbook. SciPy. <http://www.scipy.org/Cookbook/FiltFilt>.
+        """
+
+
+        if width < 2.0 or not isinstance(width, int):
```

Comment:
nm, i read the logic wrong the first time. how about adding a default of 2 or something reasonable for this?